war/src/main/js/components/dropdowns/utils.js

```diff
@@ -139,7 +140,57 @@ function generateDropdownItems(items, compact) {
   return menuItems;
 }
 
+function convertHtmlToItems(children) {
+  const items = [];
+  Array.from(children).forEach((child) => {
+    const attributes = child.dataset;
+    const type = child.dataset.dropdownType;
+
+    switch (type) {
+      case "ITEM": {
+        const item = {
+          label: attributes.dropdownText,
+          id: attributes.dropdownId,
```

Comment:
This looks like the likely culprit of a lot of `id="undefined"`?

Reply:
Fixed - good spot!